documentation/contributing/pullrequests.markdown

```diff
@@ -56,16 +54,18 @@ The ``*-devel`` branches must not be force-pushed.
 No one is allowed to directly commit to the ``*-devel`` branches of the repositories. Instead they should create feature branches and add pull-requests.
 The only commits that might be pushed directly are cherry-picks from older branches (see below) and administrative changes (e.g. CHANGELOGS, tags).
 
-### 1+1 == Merge
+### Two "Approves" == Merge
 
-If you see a trustworthy "+1" note in a request or the requestor is a fellow maintainer and all feedback has been addressed by the requestor,
-merge the request after your own review. Otherwise add a "+1" note after you are satisfied after the review.
+If you see a trustworthy approval review in a request or the requestor is a fellow maintainer and all feedback has been addressed by the requestor,
+merge the request after your own review. Otherwise submit an "Approval" review after you are satisfied after the review.
 
 ### Squash-Merge If Possible
 
 Pull-requests with a single commit should *always* be squash-merged (github supports this via a drop-down list on the "merge" button).
 If a request contains "fixup commits" it should also be squash-merged. If you are unsure this is to the requestor's liking, ask him.
 
+Note: Github recently added a [Rebase and merge](https://github.com/blog/2243-rebase-and-merge-pull-requests) option for approving pull requests - this is likely the recommended method going forward, but this has not been finalized.
+
```

Comment:
That's what you wrote in the patch I commented on:
>  Github recently added a [Rebase and merge] option for approving pull requests

That's probably just a typo.

Actually, I don't think people should be encouraged to use rebase-and-merge with MoveIt.
First of all, to the best of my knowledge it does not actually "rebase and merge" but "rebase and fast-forward".
@rhaschke pointed out some time ago that a merge commit for multiple commits is actually
a good thing: One can see where each changeset (multiple commits) entered the main branches and it is easy to perceive them as a group.
Also it's more easy for the person doing the release to collect notable changes (@130s complained about this already when I cherry-picked changes to other branches. rebase-and-merge is much worse in this regard).
I consider this quite important especially with the new huge git repo that sees *a lot* of changes.
If they are no proper group of commits, then they might as well be different pull requests, but in that case I'm fine with rebase-and-merge.
For fixup commits I still like to keep the current policy: I don't like to see them in the main repo. They add an item to the history that is totally not relevant and might even hamper bisects. So they should be squash-merged.

Are there other opinions?